### DISCUSSION TITLE
What payment method are there in Upwork

### DISCUSSION THREAD

**Discussion by goldsunshines** [Finalgoal231/support#12] [Q&A] [answered]
Hi, I am new at Upwork.
Please help me

**Comment by noahstoner** [comment, marked as answer]
Upwork accepts Visa, Discover, Diners Club, Diners Club International, Mastercard, and American Express. We also accept prepaid cards. Learn more about adding a credit or prepaid card to your account. Freelancers with a verified PayPal account can use PayPal as their billing method.
